packages/react-core/src/components/DualListSelector/examples/DualListSelectorComposable.tsx

```diff
@@ -138,17 +138,16 @@ export const DualListSelectorComposable: React.FunctionComponent = () => {
   };
 
   const buildEmptyState = (isAvailable: boolean) => (
-    <EmptyState variant={EmptyStateVariant.small}>
-      <EmptyStateIcon icon={SearchIcon} />
-      <Title headingLevel="h4" size="md">
-        No results found
-      </Title>
+    <EmptyState variant={EmptyStateVariant.sm}>
+      <EmptyStateHeader headingLevel="h4" titleText="No results found" icon={SearchIcon} />
       <EmptyStateBody>No results match the filter criteria. Clear all filters and try again.</EmptyStateBody>
-      <EmptyStatePrimary>
-        <Button variant="link" onClick={() => onFilterChange('', isAvailable)}>
-          Clear all filters
-        </Button>
-      </EmptyStatePrimary>
+      <EmptyStateFooter>
+        <EmptyStatePrimary>
```

Comment:
@srambach Should we still be using EmptyStatePrimary for anything? Or should it be replaced entirely with EmptyStateActions? FWIW our EmptyStatePrimary is just a div wrapper without any predefined classes.